AK/Time.h

```diff
@@ -54,7 +54,10 @@ unsigned day_of_week(int year, unsigned month, int day);
 // can be negative.
 constexpr int day_of_year(int year, unsigned month, int day)
 {
-    VERIFY(month >= 1 && month <= 12);
+    if (is_constant_evaluated())
+        VERIFY(month >= 1 && month <= 12); // Note that this fails to compile if and only if month is invalid.
```

Comment:
"fails to compile" -> "fails" (since constexpr is only optionally constant evaluated, as opposed to consteval)

Reply:
No, it doesn't fail. That's the weird thing I tried to document: We cannot `outln` in constexpr code. However, for some reason this only fails to compile if and only if `month >= 1 && month <= 12` evaluates to false. But it is never executed either way.

What's a better way to document this?